src/index.html

```diff
@@ -158,6 +158,7 @@
       sizes="120x120"
       href="assets/icons/apple-icon-120.png"
     />
+    <link rel="manifest" href="src/manifest.json" />
```

Comment:
could this overlap or conflict with our other (offline functionality) service worker in any way?

Reply:
I don't think it will conflict with any other (offline functionality) service worker here. What do you suggest in this?